src/edu/harvard/CS50.java

```diff
@@ -258,10 +260,10 @@ public static String getString(String prompt)
     {
         // try to get a String from user, returning null on error
         System.out.print(prompt);
-        Scanner s = new Scanner(System.in).useDelimiter("\\n|\\r|\\r\\n");
+        BufferedReader br = new BufferedReader(InputStreamReader(system.in));
```

Comment:
Unclear whether we'll include just yet, but minimally, `system.in` should be capitalized!